docker-compose.yaml

```diff
@@ -0,0 +1,39 @@
+#https://docs.docker.com/registry/deploying/#deploy-your-registry-using-a-compose-file
+services:
+  nginx:
+    image: "nginx:alpine"
+    ports:
+      - 5043:443
+    restart: unless-stopped
+    links:
```

Comment:
I don't think we need this, since our registry is named `registry` already, and will be reachable by nginx.